modules/nodes-containers-volumes-adding.adoc

```diff
@@ -81,6 +81,48 @@ For example:
 ----
 $ oc set volume dc/registry --add
 ----
++
+[TIP]
+====
+You can alternatively apply the following YAML to add the volume:
+
+.Sample deployment config with an added volume
+[%collapsible]
+=====
+[source,yaml]
+----
+kind: DeploymentConfig
+apiVersion: apps.openshift.io/v1
+metadata:
+  name: registry
+  namespace: registry
+spec:
+  replicas: 3
+  selector:
+    app: httpd
+  template:
+    metadata:
+      labels:
+        app: httpd
+    spec:
+      volumes: <1>
+        - name: volume-pppsw
+          emptyDir: {}
+      containers:
+        - name: httpd
+          image: >-
+            image-registry.openshift-image-registry.svc:5000/openshift/httpd:latest
+          ports:
+            - containerPort: 8080
+              protocol: TCP
+          resources: {}
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+          imagePullPolicy: Always
+----
```

Comment:
Why do in some cases, such as adding labels, you only include the minimum information and the parameters to add, but here and other examples (particularly the `oc set volumes` examples) you include many more parameters from the spec?

Reply:
I was cautious about removing too much.  I went back to be a little more aggressive.  I also checked by trial-and-error by diffing YAML from before and after to make sure nothing was lost.